json_schema_generator/generator.py

```diff
@@ -58,10 +58,14 @@ def to_dict(self, base_object=None, object_id=None, first_level=True, required =
         else:
             schema_dict["type"] = schema_type.json_type
 
+        if schema_dict["type"] == "object":
```

Comment:
If this check is already covered by `schema_type == ObjectType` could maybe put this under the next `if`? Not a big deal/maybe that's not quite the same thing anyway